src/base-command.ts

```diff
@@ -77,10 +77,11 @@ export default abstract class BaseCommand extends Command {
 
     // Support -- input ex. architect exec -- ls -la
     const double_dash_index = argv.indexOf('--');
+    let command: string[] = [];
     if (double_dash_index >= 0) {
-      const command = argv.slice(double_dash_index + 1, argv.length).join(' ');
+      command = argv.slice(double_dash_index + 1, argv.length);
```

Comment:
would the following work? 

```ts
argv.slice(double_dash_index + 1, argv.length).map((arg) => `'${arg}'`).join(' ');
```

Reply:
One potential issue with doing that is;

`architect exec -- sh -c "echo \"hi\"; echo 2"` is a valid input, which yields the command  `'sh' '-c' 'echo 'hi'; echo 2'`. 

If you pass that to `stringArgv` you get `[ 'sh', '-c', 'echo ', "hi'; echo 2'" ]` which is wrong, what we want is `[ 'sh', '-c', 'echo', "hi'; echo2" ]`.

I think most arbitrary ways of joining the command into a string can run into similar problems where some input exists that makes it impossible for us to go from array -> string -> array without guessing, unless we chose the poop emoji and then banned anyone from using that in the command 💩